.github/workflows/nightly-rosetta-pax-build.yaml

```diff
@@ -95,15 +95,25 @@ jobs:
       BASE_IMAGE: ${{ needs.metadata.outputs.BASE_IMAGE_AMD64 }}
     secrets: inherit
 
+  # arm64:
+  #   needs: metadata
+  #   uses: ./.github/workflows/_build_rosetta.yaml
+  #   with:
+  #     ARCHITECTURE: arm64
+  #     BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
+  #     BASE_LIBRARY: ${{ needs.metadata.outputs.BASE_LIBRARY }}
+  #     BASE_IMAGE: ${{ needs.metadata.outputs.BASE_IMAGE_ARM64 }}
+  #   secrets: inherit
   arm64:
     needs: metadata
-    uses: ./.github/workflows/_build_rosetta.yaml
-    with:
-      ARCHITECTURE: arm64
-      BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
-      BASE_LIBRARY: ${{ needs.metadata.outputs.BASE_LIBRARY }}
-      BASE_IMAGE: ${{ needs.metadata.outputs.BASE_IMAGE_ARM64 }}
-    secrets: inherit
+    runs-on: ubuntu-22.04
+    outputs:
+      DOCKER_TAG_MEALKIT: ''
```

Comment:
Just for completeness you should add: `DOCKER_TAG_FINAL`

Reply:
Added.